examples/validate.rs

```diff
@@ -8,7 +8,7 @@ pub struct MinimalHeader {
 
 // zero_digest & get_digest from https://github.com/gwihlidal/dxil-signing/blob/master/rust/src/main.rs
 
-fn zero_digest(buffer: &mut [u8]) -> () {
+fn zero_digest(buffer: &mut [u8]) {
     let buffer_ptr: *mut u8 = buffer.as_mut_ptr();
     let header_ptr: *mut MinimalHeader = buffer_ptr as *mut _;
     let header_mut: &mut MinimalHeader = unsafe { &mut *header_ptr };
```

Comment:
Clippy complains about an alignment mismatch on `*u8` vs `*MinimalHeader` (`1 < 4`). Should we change the repr of the latter to `repr(packed)`?

Reply:
Done

Reply:
The `#[repr(C)]` can/should disappear then, right?